Tests/chocolateyguicli.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+Import-Module helpers/gui-helpers
+
+Describe "chocolateyguicli" -Tag ChocolateyGuiCli {
+    BeforeDiscovery {
+        # Perhaps a better way is to pull these from the LiteDB similar to how we do features from the xml in CLI, but this will do for an initial setup.
+        $Features = (Invoke-GuiCli feature list -r).Lines | ConvertFrom-Csv -Delimiter '|' -Header Name, state, description | Select-Object Name, @{Name = 'enabled'; Expression = { $_.state -eq 'Enabled' } }

```

Comment:
Could we split this over multiple lines to make it more readable and possibly extract this to a different helper script, as it would probably be useful in multiple places in the future.

Reply:
I've moved this into a `Get-GuiFeature` function that allows to specify which feature to get, and also returns the description in case we want to add tests surrounding those.